### ISSUE TITLE
CustomItemLoader is on feature-freeze

### ISSUE THREAD

**Issue by alvin0319** [alvin0319/CustomItemLoader#37]
CustomItemLoader is now on feature-freeze. this means I will not do anything about feature-request. because it's hard to find the working NBT property and I don't have time to do PHP (due to [Kookie](https://github.com/organization/Kookie)).

**Contribution is still welcome! (even it is about feature).**

But I'll continue the protocol update as far as I can.

When I have time to do this, I'll do this again! Thank you for using my plugin.

TL;DR CustomItemLoader is on feature-freeze, there is only protocol update.
